readme.md

```diff
@@ -451,6 +451,21 @@ const limiter = rateLimit({
 })
 ```
 
+### `validation`
```

Comment:
I'm thinking about renaming this to "validate" - I think it fits better with the other method names, and it's actually what I typed here at first.

Reply:
I agree, renamed it in ee5c18a